### PR TITLE
Update the Blazor WebAssembly app name to match the server

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
We've had resolved this issue as part of https://github.com/dotnet/aspnetcore/pull/29764/files but the client project needs to be updated too.

### CODE REVIEW

**Review by pranavkm — APPROVED** [review]
Do you know where that name gets appears?

**Review by TanayParikh — APPROVED** [review]

**Comment by mkArtakMSFT** [comment]
> Do you know where that name gets appears?

In VS New Project Dialog

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "WebAssembly"
   ],
   "name": "Blazor WebAssembly App",
-  "defaultName": "WebApplication",
+  "defaultName": "BlazorApplication",
```
Comment (danroth27) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json:10]:
```suggestion
  "defaultName": "BlazorApp",
```

For consistency with https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json#L448

**Review by danroth27 — CHANGES_REQUESTED** [review]

**Comment by mkArtakMSFT** [comment]
Thanks @danroth27.
Can you sign-off now so that I can merge this?

**Review by danroth27 — APPROVED** [review]

**Pull request merged**